### PR TITLE
Sorry Operation error.

### PR DESCRIPTION
**Pull request by suixinio**
Sorry for the mistake.

### CODE REVIEW

**Comment by rnevet** [comment]
Hmm, is this a PR?

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,5 @@ nohup.out
 error.log
 *.idea
 *.csv
-*.sqlite3
```
Comment (rnevet) [inline .gitignore]: sqlite should be kept out of git.

Comment (suixinio) [reply .gitignore]: I am sorry。 I wanted to update the code, but accidentally carried out pull requests, suddenly be at a loss what to do. I don't understand what PR means. I'll pay attention to it later. I'm sorry

**Comment by rnevet** [comment]
No problem, PR = Pull request. :)

**Pull request closed without merging**